version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.40.15-SNAPSHOT"
+ThisBuild / version := "0.40.16-SNAPSHOT"
```

Comment:
Could you please revert this change? Versions are managed by the release plugin.

Reply:
Sure!, reverting the version.